pkg/controller/nodenetworkconfigurationpolicy/policyconditions/conditions.go

```diff
@@ -93,31 +93,33 @@ func Update(cli client.Client, policy *nmstatev1alpha1.NodeNetworkConfigurationP
 		}
 
 		nodes := corev1.NodeList{}
-		nodeSelectorFilter := client.MatchingLabels(policy.Spec.NodeSelector)
-		err = cli.List(context.TODO(), &nodes, nodeSelectorFilter)
+		err = cli.List(context.TODO(), &nodes)
```

Comment:
I think we should list only Nodes that are available for scheduling. If we have one faulty node in the cluster, it would keep all Policies from completing.